jenkins/opensearch/perf-test.jenkinsfile

```
@@ -177,19 +177,23 @@ pipeline {
                             postCleanup()
                         }
                         aborted {
-                            script {
-                                def stackName = "test-single-security-${BUILD_ID}-${ARCHITECTURE}-perf-test"
+                             script {
+                                stackName = "test-single-security-${BUILD_ID}-${ARCHITECTURE}-${BUILD_NUMBER}"
                                 withCredentials([string(credentialsId: 'jenkins-aws-account-public', variable: 'AWS_ACCOUNT_PUBLIC')]) {
-                                    withAWS(role: 'opensearch-test', roleAccount: "${AWS_ACCOUNT_PUBLIC}", duration: 900, roleSessionName: 'jenkins-session') {
+                                    withAWS(role: 'opensearch-test', roleAccount: "${AWS_ACCOUNT_PUBLIC}", duration: 900, roleSessionName: 'jenkins-session', region:'eu-west-1') {
                                         try {
-                                            def stack = cfnDescribe(stack: stackName)
+                                            def stack = null
+                                            try {
+                                                stack = cfnDescribe(stack: stackName)
+                                            } catch (Exception e) {
+                                                echo "Stack '${stackName}' does not exist, nothing to remove"
+                                            }
                                             if (stack != null) {
+                                                echo "Deleting stack '${stackName}'"
                                                 cfnDelete(stack: stackName, pollInterval:1000)
-                                            } else {
-                                                echo "Stack '${stackName}' does not exist, nothing to remove"
                                             }
-                                        } catch (e) {
-                                            echo "Exception occurred while deleting the cloudformation stacks" + e.toString()
+                                        } catch (Exception e) {
+                                            echo "Exception occurred while deleting the CloudFormation stack: ${e.toString()}"
```

You can remove `throw e` and error this one out
```suggestion
                                            error "Exception occurred while deleting the CloudFormation stack: ${e.toString()}"
```